src/test/java/org/opensearch/security/auditlog/integration/BasicAuditlogTest.java

```diff
@@ -132,8 +132,8 @@ public void testSSLPlainText() throws Exception {
         final List<AuditMessage> messages = TestAuditlogImpl.doThenWaitForMessages(() -> {
             final RuntimeException ex = Assert.assertThrows(RuntimeException.class,
                 () -> nonSslRestHelper().executeGetRequest("_search", encodeBasicHeader("admin", "admin")));
-            Assert.assertEquals("org.apache.http.NoHttpResponseException", ex.getCause().getClass().getName());
-        }, 4);
+            Assert.assertEquals("org.apache.hc.core5.http.NoHttpResponseException", ex.getCause().getClass().getName());
+        }, 1);
```

Comment:
Changed this to 1 to get it passing. The 4 looks arbitrary to me based on this PR: https://github.com/opensearch-project/security/pull/1758

Is there any reason to check for 4 messages?